### DISCUSSION TITLE
Siri 竟然聽得懂哈利波特的咒語

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS4719-C/Python-Community-GS4719-C#518] [General]
各位使用iPhone的哈利波特迷們 想要將手機變成魔法杖嗎？
首先，先至「設定」中把Siri的語言改成中文。
接這叫出Siri，並對他說出以下咒語，試試你的手機會有什麼反應吧！

1.Lumos
這句咒語的意思是「魔法杖發光」，換句話說，只要對著Siri說「Hey Siri Lumos」，就能啟動iPhone的手電筒。

2.Nox
和上句咒語同樣原理，不過這句的意思是「魔杖滅光」，所以只要對著Siri說「Hey Siri Nox」，iPhone手電筒就能自動關閉。

3.Accio
《哈利波特》裡的「召喚咒」，能讓遠處的目標朝自己飛來，只要對著Siri說「Hey Siri + App英文名稱」，就能觸發指令打開相應的App，例如要用Apple Pay付款時就可以說「Hey Siri Wallet」。

4.Avada Kedavra
索命咒，在《哈利波特》故事中有攻擊敵人或將敵人殺死的意思，應用在iPhone上，Siri會將手電筒的亮度調低，並讓手機進入飛航模式，有種「讓手機死亡」的意思。

5.Wingardium Leviosa
飛行咒，故事中的意思是讓事務懸浮起來，對著Siri喊「Hey Siri Wingardium Leviosa」就可以開啟航班查詢。

取自 [Siri聽得懂《哈利波特》咒語！簡單設定用「5句咒語」操作iPhone變身巫師 ｜克蘿伊星人 ]

**Comment by hapiwangy** [comment]
這真的是一個方便的功能，但如果網站連結可以是超連結的話就更好了

**Comment by leon10456** [comment]
第三個其實直接說app名稱也能做到一樣的事情喔

**Comment by PoooYen** [comment]
超好笑，只能說蘋果也太幽默了吧哈哈

**Comment by HuanYuuuuuu** [comment]
我也有玩過1.Lumos這個的 但可惜我的Siri聽不懂

**Comment by Yuhsin000** [comment]
之前在youtube short video 看過耶，搭配哈利波特的音樂超酷的!!

**Comment by sys511613** [comment]
對siri說「光守殿統」也可以把手電筒關掉哦

**Comment by Yao900108** [comment]
之前也有玩過這個，真的很可愛

**Comment by Jay-722** [comment]
哈哈，這什麼詭異功能

**Comment by stan890425** [comment]
我在KTV說Hey Siri Lumos，全部人手機就亮起來了😁

**Comment by Ben6uo** [comment]
哇塞 感覺好正 可惜我是安桌用戶

**Comment by minniewei** [comment]
可惡，我也是安卓用戶！！我也想體驗呀

**Comment by treeann** [comment]
哈利波特真的是從小愛到大

**Comment by jcpan821** [comment]
剛剛試還真的可以

**Comment by jason79461385** [comment]
剛剛試了下 用佛地魔的方式對著siri喊Avada Kedavra 結果無法 QQ

**Comment by alexthebigpp** [comment]
哈利波特的咒語本來也是作者根據拉丁文設計出來的，所以除了要說這是彩蛋外，能夠聽懂也不是那麼沒道理。

**Comment by 110409538** [comment]
好酷,但我的手機不是iphone...

**Comment by bloodtrail0511** [comment]
變身魔法師XD

**Comment by etin2468** [comment]
這不是很久之前就有的功能了嗎?

**Comment by Lynn12378** [comment]
太好玩了吧!可惜我手機是adroid的

**Comment by Jimmy911218** [comment]
哈哈感覺一定是魔法粉
